changelog_generator/__main__.py

```diff
@@ -0,0 +1,107 @@
+import os
+from typing import List, NamedTuple
+
+from jinja2 import Environment, FileSystemLoader
+
+from changelog_generator.commit import Commit
+from changelog_generator.repository_manager import RepositoryManager
+
+
+class CommitTree(NamedTuple):
```

Comment:
This is a `dataclasses.dataclass`

Don't use `typing` objects for anything else other than type hints